dashboard-visualiser-jsreport/docker-compose.yml

```diff
@@ -3,6 +3,12 @@ version: '3.9'
 services:
   dashboard-visualiser-jsreport:
     image: jsreport/jsreport:3.2.0
+    healthcheck:
+      test: wget --no-verbose --tries=1 --spider -q http://localhost:5488/ || exit 1
```

Comment:
Isn't there a healthceck endpoint for jsreport ?

Reply:
Yep you're right there is an endpoint, updated it in [533796a](https://github.com/jembi/platform/pull/191/commits/533796ad0cc26c446131335675ffb2a1fbeab4a3)